### PR TITLE
fix: add landing page github linkout

### PR DESCRIPTION
**Pull request by djmcquillan**
fixes issue #114 

- Landing page updates
- Added github logo/link
- Fixed broken links

### CODE REVIEW

**Comment by juliaxbow** [comment]
The gh icon seems to be broken
Screen Shot 2022-11-23 at 10 27 37 PM

File: User Guide.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
+[View in GitHub ![GitHub Logo](images/GitHubLogo.png#right)](https://github.com/pln-planning-tools/StarMaps)
```
Comment (SgtPooki) [inline User Guide.md:1]: We should add this link client side instead of updating the markdown directly

File: pages/index.tsx [hunk 2]
```diff
@@ -28,15 +28,15 @@ export async function getServerSideProps(context): Promise<{props: SSProps}> {
 
 const chakraUiRendererTheme: Parameters<typeof ChakraUIRenderer>[0] = {
   a: (props) => {
-    const { children } = props;
-    const link = children[0] as string;
+    const { children, href } = props;
```
Comment (SgtPooki) [inline pages/index.tsx:31]: 👍

File: pages/index.tsx [hunk 3]
```diff
@@ -28,15 +28,15 @@ export async function getServerSideProps(context): Promise<{props: SSProps}> {
...
+
     return (
-      <Link target="_blank" rel="noopener noreferrer" href={link} color="#4987BD">
+      <Link target="_blank" rel="noopener noreferrer" href={href} color="#4987BD">
         {children}
       </Link>
     );
   },
-};
```
Comment (SgtPooki) [inline pages/index.tsx:39]: We need to get eslint set up so we're not changing a bunch of lines unnecessarily between our different code settings

**Review by SgtPooki — CHANGES_REQUESTED** [review]
Lmk if it's too challenging to render the link without putting it in the actual markdown

File: User Guide.md (hunk 1 above)
Comment (djmcquillan) [reply User Guide.md]: moved out of md into index.tsx

File: pages/index.tsx [hunk 4]
```diff
@@ -15,6 +19,8 @@ interface SSProps {
   markdown: string
 }
 
+const starmapsGithubUrl = 'https://github.com/pln-planning-tools/StarMaps';
```
Comment (SgtPooki) [inline pages/index.tsx:22]: this should point to the User Guide.md directly

```suggestion
const starmapsGithubUrl = 'https://github.com/pln-planning-tools/StarMaps/blob/main/User%20Guide.md';
```

**Review by SgtPooki — CHANGES_REQUESTED** [review]

**Review by SgtPooki — APPROVED** [review]

**Pull request merged**